src/docs/sdk/performance.mdx

```diff
@@ -1,8 +1,8 @@
 ---
-title: "Guidelines for Tracing Support"
+title: "Guidelines for Performance Monitoring"
```

Comment:
Idea: if one of the problems is this page is hard to find, try adding `tags: [...]` here in the frontmatter, I heard it is picked up by Algolia search.

Reply:
OK merged, but lets follow up on this an iterate